docs/source/events/examples/form-submission.yaml

```diff
@@ -61,7 +65,25 @@ tracking:
   term: ''
   user_id: 571b05d3
   version: ''
-# Data submitted by the user
+# Data submitted by the user as part of the form submission.
+# Each component’s data can be found under the form-key configured in the form-builder.
+data:
+  city: London
+  country: GB
+  email: supporter@example.com
+  first_name: First
+  last_name: Last
+  mobile_number: '12345'
+  newsletter_subscription: radios:opt-in
+  phone_number: '67890'
+  phone_opt_in: checkbox:no-change
```

Comment:
We should probably say what the possible values of `newsletter_subscription`, `phone_opt_in`, and `post_opt_in` could be.

Reply:
That is not possible. Any (web)form component can have these form-keys so it could have any data.

Reply:
In that case, can we say where these values would be set?